chapters/02-estrutura-do-programa.md

```diff
@@ -111,60 +111,46 @@ Embora eu tenha afirmado que nomes de variáveis não podem conter pontos, `cons
 
 ## Retornando Valores
 
-Mostrar uma caixa de diálogo ou escrever texto na tela é um efeito colateral. Muitas funções são úteis por causa dos efeitos que elas produzem. É também possível para uma função produzir um valor, no caso dela não ser necessário um efeito colateral. Por exemplo, temos a função `Math.max`, que pega dois números e retorna o maior entre eles:
+Mostrar uma caixa de diálogo ou escrever texto na tela é um efeito colateral. Muitas funções são úteis por causa dos efeitos que elas produzem. É também possível para uma função produzir um valor, no caso de não ser necessário um efeito colateral. Por exemplo, temos a função `Math.max`, que pega dois números e retorna o maior entre eles:
 
 ```javascript
-
 console.log(Math.max(2, 4));
-
 ```
 
-Quando uma função produz um valor, é dito que ela *retorna* (return) ele. Por coisas que produzem valores sempre serem expressões no JavaScript, funções chamadas podem ser usadas como parte de uma grande expressão:
+Quando uma função produz um valor, é dito que ela *retorna* (`return`) ele. Por coisas que produzem valores serem sempre expressões no JavaScript, chamadas de função podem ser usadas como parte de uma grande expressão:
```

Comment:
""Por coisas que produzem valores serem sempre expressões no JavaScript, chamadas de função podem ser usadas como parte de uma grande expressão:"

**Original:** Anything that produces a value is an expression in JavaScript, which means function calls can be used within larger expressions. For example, the function Math.max takes any number of number values and gives back the greatest.

**Sugestão:** Tudo que produz um valor é uma expressão em JavaScript, o que significa que chamadas de função podem ser usadas dentro de expressões maiores. Por exemplo , a função `Math.max` que recebe qualquer valor que seja número e devolve o maior.